codecov_cli/commands/upload.py

```diff
@@ -164,6 +164,14 @@ def _turn_env_vars_into_dict(ctx, params, value):
         default="coverage",
         type=click.Choice(["coverage", "test_results"]),
     ),
+    click.option(
+        "--network-filter",
+        help="Specify a filter on the files listed in the network section of the Codecov report. Useful for upload-specific path fixing",
```

Comment:
I think it would be good to clarify here that the filter checks if the file path starts with the filtered string